src/pages/packer/guides/[[...page]].tsx

```diff
@@ -0,0 +1,26 @@
+import packerData from 'data/packer.json'
+import { Product } from 'types/products'
+import DocsView from 'views/docs-view'
+import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
+import { getStaticGenerationFunctions } from 'layouts/sidebar-sidecar/server'
+
+const basePath = 'guides'
+const baseName = 'Guides'
+const product = packerData as Product
+const mainBranch = 'master'
+
+const PackerGuidePage = ({ mdxSource }): React.ReactElement => {
+  return <DocsView {...mdxSource} />
+}
+
+const { getStaticPaths, getStaticProps } = getStaticGenerationFunctions({
+  product,
+  basePath,
+  baseName,
+  mainBranch,
+})
+
+PackerGuidePage.layout = SidebarSidecarLayout
+
+export { getStaticProps, getStaticPaths }
+export default PackerGuidePage
```

Comment:
Nit suggestion for consistency

```suggestion
const PackerGuidesPage = ({ mdxSource }): React.ReactElement => {
  return <DocsView {...mdxSource} />
}

const { getStaticPaths, getStaticProps } = getStaticGenerationFunctions({
  product,
  basePath,
  baseName,
  mainBranch,
})

PackerGuidePages.layout = SidebarSidecarLayout

export { getStaticProps, getStaticPaths }
export default PackerGuidesPage
```